README.md

```diff
@@ -107,19 +123,5 @@ You will see `Jekyll` detect your new files and regenerate the blog files.  You
             ...done in 121.8705398 seconds.
 ```
 
-### Restarting the container
```

Comment:
Is there a reason we want to remove this section about restarting the container?